src_c/base.c

```diff
@@ -192,6 +192,34 @@ pg_CheckSDLVersions(void) /*compare compiled to linked*/
                      linked.major, linked.minor, linked.patch);
         return 0;
     }
+    else if (linked.major == 2 && linked.minor == 0 && linked.patch < 14 &&
+             compiled.patch >=
+                 14) {  // major and minor versions match, check edge case
+        /* SDL 2.0.14 replaces some macros with symbols, see
+         * https://github.com/libsdl-org/SDL/commit/316ff3847b4d9d87d9b0aab15321461db0e8ae0b
+         */
+        PyErr_Format(PyExc_RuntimeError,
+                     "Known SDL incompatibility detected! (compiled with "
+                     "version %d.%d.%d, linked to %d.%d.%d)",
+                     compiled.major, compiled.minor, compiled.patch,
+                     linked.major, linked.minor, linked.patch);
+        return 0;
+    }
```

Comment:
Is this block required? My understanding is that it is merely a special case of the more generic error raised in the next `else if` block.

Reply:
Yes, it is a special case. It is the special case that has been *known* to cause a problem and reported in the wild, but other downgrades will be problematic too.

Reply:
The original PR was *only* the special case.

Reply:
Now that I think about it, SDL 2.0.5 also has a lot of incompatibilities, but 2.0.5 is ancient